docs/content/dagster-plus/managing-deployments/branch-deployments/change-tracking.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: "Tracking asset changes in Dagster+ Branch Deployments | Dagster Docs"
+description: "Change tracking in Branch Deployments makes it eaiser for you and your team to identify how changes in a pull request will impact data assets."
+---
+
+# Tracking asset changes in Dagster+ Branch Deployments
+
+<ApplicableDagsterProduct />
+
+Change tracking in Branch Deployments makes it eaiser for you and your team to identify how changes in a pull request will impact data assets. By the end of this guide, you'll understand how change tracking works and what types of asset changes can be detected.
+
+---
+
+## How it works
+
+Branch Deployments compare asset definitions in the branch deployment against the asset definitions in the main deployment. The UI will then mark changed assets accordingly. For example, if the pull request associated with the branch deployment adds a new asset, the UI will display a label indicating the addition.
+
+You can also apply filters to show only new and changed assets in the UI. This makes it easy to understand which assets will be impacted by the changes in the branch deployment.
+
+---
+
+## Supported change types
+
+Change tracking can detect the following changes to assets:
+
+- [New assets](#new-assets)
+- [Code versions](#code-versions)
+- [Upstream dependencies](#upstream-dependencies)
+- [Partitions definitions](#partitions-definitions)
+
+### New assets
+
+If an asset is new in the branch deployment, the asset will have a **New in branch** label in the asset graph:
+
+<Image
+src="/images/dagster-cloud/managing-deployments/change-tracking-new.png"
+width={764}
+height={308}
+/>
+
+### Code versions
+
+Change tracking can detect when an asset's `code_version` has changed, whether it's just been added, removed, or updated.
```

Comment:
I forgot to include this in my initial draft, but there is a subtle difference in the type of `code_version` we detect for change tracking. A user can specify a `code_version` argument on the asset decorator/definition (what's shown in this example). but Dagster will also automatically compute a code version for each materialization of an asset if there is no user specified code version. However - we only detect the one on the decorator/definition and NOT the one that is computed and attached to a materialization. I'm not sure how to best communicate that distinction here. Maybe something like 

"Note: Change tracking only detects `code_version`s that are attached to the asset decorator" ??

Reply:
Oh good call - I forgot about this. I'll see what I can do!

Reply:
just bumping this comment in case it got lost. I don't think this is a huge huge deal to get in right away but it definitely could be confusing